### PR TITLE
[CDS-78591]: adjust schema

### PR DESCRIPTION
**Pull request by tmacari-harness**
[CDS-78591]: adjust schema

[CDS-78591]: https://harness.atlassian.net/browse/CDS-78591?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: v0/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ allOf:
           type: string
       - type: string
         pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-        minLength: 1
+        minLength: 1add
```
Comment (prashantsharma04) [inline v0/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml:18]: is this a valid change?

File: v1/pipeline.json [hunk 2]
```diff
@@ -16181,6 +16181,7 @@
               "$ref" : "#/definitions/pipeline/common/StepSpecType"
             }, {
               "type" : "object",
+              "required" : [ "connectorRef", "image" ],
```
Comment (prashantsharma04) [inline v1/pipeline.json:16184]: will this not break existing pipeline?

**Review by utkarshchoubey-harness — CHANGES_REQUESTED** [review]

File: v1/pipeline.json (hunk 2 above)
Comment (utkarshchoubey-harness) [reply v1/pipeline.json:16184]: this is v1, it shouldn't impact anything (Its not used anywhere)

File: v0/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml (hunk 1 above)
Comment (tmacari-harness) [reply v0/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml:18]: not sure how it get there, will fix

**Review by utkarshchoubey-harness — APPROVED** [review]

File: v1/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml [hunk 3]
```diff
@@ -52,6 +59,9 @@ allOf:
       - type: string
 $schema: http://json-schema.org/draft-07/schema#
 type: object
+required:
```
Comment (prashantsharma04) [inline v1/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml:62]: will this not break existing pipelines

Comment (tmacari-harness) [reply v1/pipeline/steps/common/aws-cdk-bootstrap-step-info.yaml:62]: there are no existing pipelines, feature is still under development.

**Review by prashantsharma04 — APPROVED** [review]

**Pull request merged**